tests/unit/lms/security_test.py

```diff
@@ -20,19 +21,33 @@
 
 
 class TestIncludeMe:
-    def test_it_sets_security_policy(self, pyramid_config, SecurityPolicy):
+    def test_it_sets_security_policy(
+        self, pyramid_config, SecurityPolicy, PathPolicyGetter
+    ):
         includeme(pyramid_config)
 
-        SecurityPolicy.assert_called_once_with()
+        PathPolicyGetter.assert_called_once_with(
+            path_policy_mapping={
+                "/admin": LMSGoogleSecurityPolicy,
+                "/googleauth": LMSGoogleSecurityPolicy,
+                # Fallback for the rest
+                "/": LTISecurityPolicy,
+            }
+        )
+        SecurityPolicy.assert_called_once_with(PathPolicyGetter.return_value)
         assert (
             pyramid_config.registry.queryUtility(ISecurityPolicy)
             == SecurityPolicy.return_value
         )
 
-    @pytest.fixture(autouse=True)
+    @pytest.fixture()
```

Comment:
What's the reason for removing `autouse=True` here? I'd guess that we never want any of `includeme()`'s unit tests to accidentally be using the real `SecurityPolicy` to `autouse` probably makes sense